From 63dd59fd3207221f94aeb303b93a371a36de416c Mon Sep 17 00:00:00 2001
From: Mianhan Liu <liumh1@shanghaitech.edu.cn>
Date: Thu, 30 Sep 2021 23:10:17 +0800
Subject: [PATCH] net/bluetooth/msft.c: remove superfluous headers

msft.c hasn't use any macro or function declared in mgmt_util.h and net/bluetooth/mgmt.h.
Thus, these files can be removed from msft.c safely without
affecting the compilation of the ./net module
---
 net/bluetooth/msft.c | 2 --
 1 file changed, 2 deletions(-)

diff --git a/net/bluetooth/msft.c b/net/bluetooth/msft.c
index 255cffa55..a91179ce2 100644
--- a/net/bluetooth/msft.c
+++ b/net/bluetooth/msft.c
@@ -5,10 +5,8 @@
 
 #include <net/bluetooth/bluetooth.h>
 #include <net/bluetooth/hci_core.h>
-#include <net/bluetooth/mgmt.h>
 
 #include "hci_request.h"
-#include "mgmt_util.h"
 #include "msft.h"
 
 #define MSFT_RSSI_THRESHOLD_VALUE_MIN		-127
-- 
2.25.1

